packages/components/src/components/hds/pagination/nav/arrow.ts

```diff
@@ -6,12 +6,36 @@
 import Component from '@glimmer/component';
 import { action } from '@ember/object';
 import { assert } from '@ember/debug';
+import type { HdsPaginationDirections } from '../types.ts';
+import type { HdsInteractiveSignature } from '../../interactive/index.ts';
+import type { IconName } from '@hashicorp/flight-icons/svg';
 
-export const DIRECTIONS = ['prev', 'next'];
+export const DIRECTIONS: HdsPaginationDirections[] = ['prev', 'next'];
```

Comment:
can we convert this into an enum and move it into `types.ts`? we have a pattern around such constants